### PR TITLE
tests CustomUserManager class

### LINKED ISSUE #46: #17: Cover with tests CustomUserManager class
1) Read guideline [here](https://gist.github.com/iamnovichek/801ac395a06a7d8b04b022084ace684d)
2) Create `test_models.py` in `tracker/tests/`
3) Write tests for `CustomUserManager` class from `tracker/models.py`
4) Provide some screenshots to prove that your tests work

### PR DESCRIPTION
**Pull request by ikemHood**
closes #46 
 
 - tested create_user and create_superuser
 
 
Screenshot from 2024-11-30 19-10-51

### CODE REVIEW

File: tracker/models.py [hunk 1]
```diff
@@ -21,13 +22,26 @@ class CustomUserManager(BaseUserManager):
...
         """
         Creates a new user
-        :param email: str = None
-        :param password: str = None
+        :param email: str
+        :param password: str
+        :param role: str = None
         :return: CustomUser
         """
+        if not email:
```
Comment (iamnovichek) [inline tracker/models.py]: For `email` and `password` validations don't make any sense because the python side will crash

**Review by iamnovichek — CHANGES_REQUESTED** [review]

**Comment by iamnovichek** [comment]
@ikemHood Core review done

**Comment by ikemHood** [comment]
Ok, would still be adding some more test shortly

File: tracker/models.py (hunk 1 above)
Comment (ikemHood) [reply tracker/models.py]: hmmm, ok.

should I validate email if available?

Comment (iamnovichek) [reply tracker/models.py]: yep you could

**Comment by ikemHood** [comment]
Screenshot from 2024-11-30 22-01-42

**Comment by ikemHood** [comment]
Please review.

**Review by iamnovichek — APPROVED** [review]

**Pull request merged**